apps/client/src/components/Location/LocationResult.js

```diff
@@ -62,13 +62,14 @@ const LocationResult = ({
 
       {isActive("locationResult") && (
         <Nav
+          formEnds={!hasSTTR}
+          nextText={!hasSTTR ? "Naar het omgevingsloket" : "Naar de vragen"}
```

Comment:
Can you remove the ! and change the order of the ternary options?

Reply:
Better!